pkg/subctl/cmd/gather/gather.go

```diff
@@ -116,54 +116,46 @@ func gatherData(cluster *cmd.Cluster) bool {
 		}
 	}
 
-	gatherDataByCluster(cluster, directory)
+	status := cli.NewReporter()
+	status.Start("Gathering information from cluster %q\n", cluster.Name)
+	gatherDataByCluster(cluster, directory, status)
 
-	fmt.Printf("Files are stored under directory %q\n", directory)
+	status.Success("Files are stored under directory %q\n", directory)
+	status.End()
```

Comment:
I don't think any of this needs to change. L123 is an informational message and does not need to be deemed as Success so leave it as `fmt.Printf`.  Same with L120 - since we do Start/End for each module/type below we can keep this as informational (ie `fmt.Printf`) and keep it in `gatherDataByCluster`. There is no need to create a status here and pass into `gatherDataByCluster`.